### PR TITLE
Documenting recent operator functionality.

### PR DESCRIPTION
**Pull request by jmencak**
Documentation for:
  - profile settings rollback on profile changes
  - the default operator's CR being overwritten
  - a complete example of using custom profiles

Minor fixes and improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,13 @@ specification](#custom-tuning-specification) is passed to all
 daemons running in the cluster in the format that the daemons understand.
 The daemons run on all nodes in the cluster, one per node.
 
+Node-level settings applied by the containerized tuned are rolled back on
+an event that triggers a profile change or when the containerized tuned
+daemon is terminated gracefully by receiving and handling a termination
+signal.
```
Comment (ashishkamra) [inline README.md]: terminated gracefully by "sending a termination signal"

File: README.md [hunk 2]
```diff
@@ -16,8 +16,13 @@ specification](#custom-tuning-specification) is passed to all
 daemons running in the cluster in the format that the daemons understand.
 The daemons run on all nodes in the cluster, one per node.
 
+Node-level settings applied by the containerized tuned are rolled back on
+an event that triggers a profile change or when the containerized tuned
```
Comment (ashishkamra) [inline README.md]: I guess I am little confused on what the settings are rolled back to ?

File: README.md [hunk 3]
```diff
@@ -67,7 +79,7 @@ The `profile:` section lists tuned profiles and their names.
 
       [sysctl]
```
Comment (ashishkamra) [inline README.md]: Is the [sysctl] part of the profile data optional? If the user wants to create a custom CR based on an existing tuned profile such as sap-hana, can they only specify the "include=sap-hana" directive under [main] and skip specifying the [sysctl] part?

File: README.md (hunk 1 above)
Comment (jmencak) [reply README.md]: Sending?  The containerized tuned daemon itself doesn't send any termination signals.  It *receives* and *handles* termination signals.  openshift-tuned though does send a `TERM` signal to tuned when it receives a `TERM` signal itself.

File: README.md (hunk 2 above)
Comment (jmencak) [reply README.md]: All the settings changed by a profile (e.g. sysctls) are rolled back upon application of a new profile.  Say profile A sets `net.ipv4.ip_local_port_range="1024 65535"`.  Before profile A is set though, tuned daemon looks at `net.ipv4.ip_local_port_range` and sees the default value `32768 60999`.  It remembers that value and upon a change to profile B `net.ipv4.ip_local_port_range` is unapplied/rolled-back to `32768 60999`.

File: README.md (hunk 3 above)
Comment (jmencak) [reply README.md]: Yes, that's is correct.  The `[sysctl]` plugin is optional.

**Comment by jmencak** [comment]
@sjug, PTAL, so we can get this merged.

**Pull request merged**